patch_sm/Looper/Looper.cpp

```diff
@@ -0,0 +1,85 @@
+#include "daisy_patch_sm.h"
+#include "daisysp.h"
+
+using namespace daisy;
+using namespace patch_sm;
+using namespace daisysp;
+
+DaisyPatchSM patch;
+Switch       button;
+
+#define kBuffSize 48000 * 60 // 60 seconds at 48kHz
+
+// Loopers and the buffers they'll use
+Looper              looper_l;
+Looper              looper_r;
+float DSY_SDRAM_BSS buffer_l[kBuffSize];
+float DSY_SDRAM_BSS buffer_r[kBuffSize];
+
+// gain factors for input and looper
+float loop_level, in_level;
```

Comment:
Are these global for any particular reason? Looks like they could just be on the stack within the callback.